content/docs/guides/go-application.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Instrumenting a Go application
+---
+
+# Instrumenting a Go application for Prometheus
+
+Prometheus has an official [Go client library](https://github.com/prometheus/client_golang) that you can use to [instrument](/docs/instrumenting/pushing) Go applications. In this guide, we'll create a simple Go application that exposes Prometheus metrics via HTTP.
+
+NOTE: For comprehensive API documentation, see the [GoDoc](https://godoc.org/github.com/prometheus/client_golang) for Prometheus' various Go libraries.
+
+## Installation
+
+You can install the main `prometheus` library using [`go get`](https://golang.org/doc/articles/go_command.html):
+
+```bash
+go get github.com/prometheus/client_golang/prometheus
+```
+
+See the [GoDoc](https://godoc.org/github.com/prometheus/client_golang#pkg-subdirectories) for a listing of other libraries you may need to install.
+
+## How Go instrumentation works
```

Comment:
This is exposition, not instrumentation. It's fine to do it first, but let's get the terminology right